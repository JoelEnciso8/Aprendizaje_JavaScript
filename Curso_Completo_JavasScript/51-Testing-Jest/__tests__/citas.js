import  Citas  from "../js/classes/Citas";
/*
probamos el metodo citas en un Snapshot para tener la evidencia visual de como se implementa el codigo y ver su funcionamiento
*/ 



describe('Probar la clase de citas',()=>{

    const citas = new Citas()

    test('Agregando nueva cita',()=>{
        const citaObj = {
            mascota: 'hook',
            propietario: 'Joel',
            telefono: '1345',
            fecha: '12-12-2000',
            hora:'12:00',
            sintomas: 'sleep'
        };

        citaObj.id = Date.now()

        citas.agregarCita(citaObj)


        expect(citas).toMatchSnapshot();
    });




}); 

