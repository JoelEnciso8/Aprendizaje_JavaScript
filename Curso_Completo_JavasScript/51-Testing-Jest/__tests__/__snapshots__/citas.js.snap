// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probar la clase de citas Agregando nueva cita 1`] = `
Citas {
  "citas": [
    {
      "fecha": "12-12-2000",
      "hora": "12:00",
      "id": 1721270395130,
      "mascota": "hook",
      "propietario": "Joel",
      "sintomas": "sleep",
      "telefono": "1345",
    },
  ],
}
`;
